CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera/CameraViewController.swift

```diff
@@ -89,12 +92,20 @@ final class CameraViewController: UIViewController {
         }
 
         self.cameraLensSwitcherView.delegate = self
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(configureCameraPanesBasedOnOrientation),
+                                               name: UIDevice.orientationDidChangeNotification,
+                                               object: nil)
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
+    deinit {
+        NotificationCenter.default.removeObserver(self)
```

Comment:
[Clarification]: @igor-gini Does it make sense to unsubscribe from particular notification here using `name: UIDevice.orientationDidChangeNotification`?